content/trial_getting_started/examples/aws_hello_world_example.md

```diff
@@ -113,24 +121,12 @@ _Tip: To check out some more commands to use with Cloudify Manager, run `cfy --h
 
 An even easier way to review your deployment is through the Cloudify management console. Login to the UI and browse to the Deployments page. Select the deployment (cloudify-hello-world-example-master.aws) and explore the topology, inputs, outputs, nodes, and logs.
 
-TBD ADD picture
-
-TBD ADD picture
-
-TBD ADD picture
-
-TBD ADD picture
-
-TBD ADD picture
-
-TBD ADD picture
-
-TBD ADD picture
+![aws_hello_world_deployment_topology.png]( /images/trial_getting_started/aws_hello_world_deployment_topology.png )
 
 This will also be a good time to examine the Cloudify blueprint used in the example. The blueprint can be examined in the Cloudify UI, however in this case we will go to the Cloudify examples repository in github and examine it there: [https://github.com/cloudify-community/blueprint-examples/blob/master/hello-world-example/aws.yaml](https://github.com/cloudify-community/blueprint-examples/blob/master/hello-world-example/aws.yaml)
 
 
-## Step 5: OK, I am done, how do I tear it down?
+## Step 6: OK, I am done, how do I tear it down?
 
 To remove the deployment from AWS simply run the uninstall command:
 ```bash
```

Comment:
There is an error in the "bash code - in each example we should select just the relevant infra:
cfy uninstall cloudify-hello-world-example-master.<aws/gcp/azure/openstack>